### PR TITLE
Add 2022-09-20 newsletter

### PR DESCRIPTION
**Pull request by ptheywood**
Adds September 2022 newsletter, based on contributions via the slack channel.

### CODE REVIEW

File: _newsletters/2022-09-20-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+---
+## The University of Sheffield Research Software Engineering Community Newsletter September 2022
+
+Welcome to this month's newsletter for the research software community at The University of Sheffield, featuring news, opportunities, events and training for you.
+
+### News, Web & Blogs
+
+* [White Rose Open Educational Resources (OER) Toolkit](https://sites.google.com/york.ac.uk/oertoolkit/)
+* [GitHub - SSH commit verification now supported](https://github.blog/changelog/2022-08-23-ssh-commit-verification-now-supported/)
+* [Research IT Manchester - Meet Our Year in Industry Students!](https://research-it.manchester.ac.uk/news/2022/07/29/meet-our-year-in-industry-students/)
```
Comment (Robadob) [inline _newsletters/2022-09-20-newsletter.md:21]: Privacy error, https not supported. Not sure you can do much about it, but may upset anyone following the link.

File: _newsletters/2022-09-20-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+* [rstudio::conf(2022)](https://www.rstudio.com/conference/) talks are [available on-demand](https://www.rstudio.com/conference/2022/2022-conf-talks/).
+* [Python Package Index - targetted phishing campagin](https://twitter.com/pypi/status/1562442188285308929)
```
Comment (Robadob) [inline _newsletters/2022-09-20-newsletter.md]: Typo `campagin`

Comment (Robadob) [reply _newsletters/2022-09-20-newsletter.md]: Also kinda sad they don't have a proper newspost on pypi website about it (I looked, all I found as a giveaway of security keys for project maintainers).

File: _newsletters/2022-09-20-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+* [Python Packaging User Survey](https://www.surveymonkey.co.uk/r/M5XKQCT). The objective of this survey is to help the Python Software Foundation (PSF), Packaging Working Group and Python Packaging Authority (PyPA) better understand the challenges and opportunities within the Python packaging ecosystem. Closes 2022-09-30.
+* The Sheffield RSE webpages on [Researcher Programming Resources](https://rse.shef.ac.uk/training/programming) have been updated to include a number of new additional resources.
+
+### Events
+
+* [NVIDIA GPU Techonology Conference (#GTC22)](https://www.nvidia.com/gtc/?ncid=ref-dli-435814) - Join NVIDIA #GTC22 online (2022-09-19 to 2022-09-22) to discover the innovations that will impact your life’s work. Explore over 200 sessions from some of the brightest minds solving the world’s greatest challenges. Register for free now.
+* [Hacktoberfest 2022](https://hacktoberfest.com/) - Hacktoberfest is DigitalOcean’s annual event that encourages people to contribute to open source throughout October. New for 2022 Low or Non-Code contributions are being encouraged!
```
Comment (Robadob) [inline _newsletters/2022-09-20-newsletter.md]: Can't actually sign up for another week. Maybe add a note about that?

File: _newsletters/2022-09-20-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+* [N8 CIR Machine Learning Theme Launch](https://n8cir.org.uk/events/machine-learning-theme-launch/) - 2022-11-01, The University of Leeds. Open to all researchers, PGRs and anyone involved in Machine Learning at an N8 institution, this event will bring the Machine Learning community from across N8 together to find out more about N8 CIR, plans for the theme and all-important networking opportunities.
+* [PyData Global 2022](https://pdg22.wpengine.com) - 2022-12-01 to 2022-12-03, Online. The PyData Global conference is an opportunity to listen, learn, and share knowledge about best practices, new approaches, and emerging technologies for data management, processing, analytics, and visualization.
+
+### Training
+
+#### Research IT Training
+
+Research IT is providing a place for beginners or advanced users to expand their knowledge of HPC and different programming languages. The courses are part of the Doctoral Development Programme. For more information please visit our training registration web page (via VPN): [https://crs.shef.ac.uk](crs.shef.ac.uk).
```
Comment (Robadob) [inline _newsletters/2022-09-20-newsletter.md]: Inside the Markdown version of this on GitHub this link goes to `https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/blob/newsletter-sep-22/_newsletters/crs.shef.ac.uk`. Protocol probably wants adding.

File: _newsletters/2022-09-20-newsletter.md (hunk 1 above)
Comment (ptheywood) [reply _newsletters/2022-09-20-newsletter.md:21]: Not much I can do about Manc's SSL certs expiring I'm afraid (expired on Sunday). I assume it'll be renewed in the near future (it was issued by lets encrypt so supprised it wasn't automatically updated)

**Review by Robadob — CHANGES_REQUESTED** [review]
Added several minor comments.

Noticed that you're super inconsistent with how dates are formatted.

`2022-09-30` for News and Events sections
`3rd-4th October 2022` for training

`25 Oct 2022`, `3rd October`, `16:00 on 4 Oct 2022`, `14th of December 2022, 17:00 CET` for funding

May want to pick a single style and run with it throughout.

Otherwise looks thorough

File: _newsletters/2022-09-20-newsletter.md (hunk 3 above)
Comment (ptheywood) [reply _newsletters/2022-09-20-newsletter.md]: `Registration begins 2022-09-26`

File: _newsletters/2022-09-20-newsletter.md (hunk 4 above)
Comment (ptheywood) [reply _newsletters/2022-09-20-newsletter.md]: this was inherrited from a prior newsletter so I'd not checked it. The two parts of the md link are just the wrong way around.

**Comment by ptheywood** [comment]
Fair comment on date inconsistency, I used 8601 when I wrote the content, but other forms when it was copy pasted from elsewhere. Will standardise throughout.

**Comment by Robadob** [comment]
> I used 8601 when I wrote the content

Yeah assumed that was your doing, it's not particularly standard in media though it should be.

**Review by Robadob — APPROVED** [review]
Looks fine to me.

**Pull request merged**